### ISSUE TITLE
Adding Divi support

### ISSUE THREAD

**Issue by SteveJonesDev** [equalizedigital/accessibility-checker#73]
I installed the plugin on MountainTRAX https://mountaintraxrail.com/ which is built with Divi.  Here's what I noticed:

### Color Contrast
On the home page, this section flags the heading as a color contrast error but not the paragraph text:

![full-2](https://user-images.githubusercontent.com/2895788/204643678-5f9dbf07-4c4d-4caf-8bb5-3207b75c7fb7.png)

The heading has the font color inlined because I set it in the block (or whatever Divi calls them) for that heading.  The paragraph's color is set on on the div that contains the heading and paragraphs and is added with CSS like this:
```
.et_pb_text_2.et_pb_text {
    color: #FFFFFF!important;
}
```

All  light blue buttons should have a color contrast flag too.

![full-3](https://user-images.githubusercontent.com/2895788/204643769-8242235b-f0f6-427a-9bb4-84926e59809b.png)

Interestingly, WAVE sees the paragraph text as a color contrast error but not the heading or button, so that just goes to show how finicky this is.

Elsewhere on the page, the footer also has a contrast issue:

![full-4](https://user-images.githubusercontent.com/2895788/204643818-bb2479ba-261e-4d32-b05d-0fa1595f9793.png)

### Empty link
The play button on the video is an empty link, but the plugin does not catch this. 

![full-5](https://user-images.githubusercontent.com/2895788/204643953-d4662a76-5bbb-49a8-9d2e-55009f34e7f0.png)

```
<div class="et_pb_video_overlay_hover">
<a href="#" class="et_pb_video_play"></a>
</div>
```

### Doesn't flag A video is present
Here's the code for that video:
```
<div class="et_pb_module et_pb_video et_pb_video_0">
				
				
				<div class="et_pb_video_box">
				<video controls="">
					<source type="video/mp4" src="https://secureservercdn.net/104.238.71.109/6kc.a22.myftpupload.com/wp-content/uploads/2020/06/poudre-1.mp4">
					
				</video></div>
				<div style="background-image:url(https://secureservercdn.net/104.238.71.109/6kc.a22.myftpupload.com/wp-content/uploads/2020/06/Screen-Shot-2020-06-04-at-9.19.42-AM.png)" class="et_pb_video_overlay"><div class="et_pb_video_overlay_hover"><a href="#" class="et_pb_video_play"></a></div></div>
			</div>
```

### Incorrect Heading Order

On the contact page, there is an H2 followed by an H4.  This is not flagging n incorrect heading order.

![full-6](https://user-images.githubusercontent.com/2895788/204644104-f2cd02a5-a5db-4f01-b72d-d42b9a828109.png)

**Comment by SteveJonesDev** [comment]
Color contrast fixed with v1.7.0.

**Comment by SteveJonesDev** [comment]
Several of these issues should be resolved with the latest version of the plugin. Close due to age. New individual issues can be opened after re-testing.
